### PR TITLE
feat: Backout dynamic address book change for release 0.51

### LINKED ISSUE #13265: Back out protobuf-changes related to dynamic address book
As we postponed the dynamic address book to release 0.52, we need to back out all changes that were already committed to the protobuf repository.

Modified this for hedera-services code back out. Created issue #357 for hedera-protobufs back out.

### PR DESCRIPTION
**Pull request by iwsimon**
**Description**:

**Related issue(s)**:

Fixes #13265

### CODE REVIEW

File: hapi/build.gradle.kts [hunk 1]
```diff
@@ -28,7 +28,8 @@ tasks.cloneHederaProtobufs {
     // uncomment below to use a specific tag
     //    tag = "v0.50.0-release"
     // uncomment below to use a specific branch
-    branch = "main"
+    //    branch = "main"
+    branch = "13265-backout-dab-change"
```
Comment (kimbor) [inline hapi/build.gradle.kts:32]: I guess your intention is to merge this PR first, then PR #358 in hedera-protobufs (but don't delete associated branch) then do another PR to update the branch label here back to main.

**Review by kimbor — APPROVED** [review]

File: hapi/build.gradle.kts (hunk 1 above)
Comment (iwsimon) [reply hapi/build.gradle.kts:32]: Yes, that is the plan.

**Review by kimbor — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM!

**Review by jeromy-cannon — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]
